### PR TITLE
For backward compatibility of the edgecore trigger the upgrade command

### PR DESCRIPTION
**Pull request by WillardHu**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

If the edgecore version is earlier then v1.16, It still upgrade with the command `keadm upgrade ...`. So we need to think about backward compatibility for that. If the upgrade command no three-level command, keadm upgrades the edge components.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by WillardHu** [comment]
cc @ZhengXinwei-F @Shelley-BaoYue @fisherxu

File: keadm/cmd/keadm/app/cmd/edge/upgrade.go [hunk 1]
```diff
@@ -342,7 +342,7 @@ type Upgrade struct {
 	Reason string
 }
 
-func addUpgradeFlags(cmd *cobra.Command, upgradeOptions *UpgradeOptions) {
+func AddUpgradeFlags(cmd *cobra.Command, upgradeOptions *UpgradeOptions) {
```
Comment (ZhengXinwei-F) [inline keadm/cmd/keadm/app/cmd/edge/upgrade.go:345]: Please provide global function comments.

Comment (WillardHu) [reply keadm/cmd/keadm/app/cmd/edge/upgrade.go:346]: Done, thanks.

**Comment by ZhengXinwei-F** [comment]
lgtm

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @Shelley-BaoYue

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Comment by fisherxu** [comment]
Plz cherry-pick to release-1.16, thanks @WillardHu

**Comment by WillardHu** [comment]
> Plz cherry-pick to release-1.16, thanks @WillardHu

Done, thanks.

**Pull request merged**